cli-core/reportGlobal.js

```diff
@@ -132,6 +138,21 @@ async function create_global_report(reports,options){
     };
     
     if (progressBar) progressBar.tick()
+
+    // If an error occured during the axios call, we log it after the progress bar riched 100% (for visual purpose)
+    if (ipApiError) {
+        console.log("WARN - Unable to connect to the API http://ip-api.com (default value used for isMobile is " + isMobile + "), error : ");
```

Comment:
Is it really necessary to have so many lines of code to log the error associated to a secondary feature? Something like this could be enough:

```js
console.error(`WARN - Unable to get the connection type from https://ip-api.com (default value for isMobile will be: ${isMobile}): ${error}`)
```